packages/cli/src/commands/bundle/saveAssets.ts

```diff
@@ -30,6 +30,12 @@ function saveAssets(
     return Promise.resolve();
   }
 
+  if (!fs.existsSync(assetsDest)) {
```

Comment:
hi @thecodrr @thymikee @grabbou,
I have a situation that I need to run bundle and copy assets into new created folder based on `assetsDest` and I was relying on https://github.com/react-native-community/cli/pull/609/files#diff-e59437386ac414095e6ed43a3da37303R93 to create folder if not exists.
Any specific reason that it needs to check `assetsDest` first before copying assets?

Reply:
Since we are calling `mkdirp` in the copy function, maybe this is no longer needed?

Reply:
yes, i think it should be safe using `mkdirp` before copy files

Reply:
this check can be safely removed then. What do you think @thymikee?

Reply:
Please do

Reply:
Thanks @thecodrr @thymikee will open the PR